boot/zephyr/include/mcuboot_config/mcuboot_config.h

```diff
@@ -108,6 +108,12 @@
 
 #endif /* CONFIG_SINGLE_APPLICATION_SLOT */
 
+#ifdef CONFIG_SINGLE_APPLICATION_SLOT_RAM_LOAD
+#define MCUBOOT_RAM_LOAD 1
```

Comment:
And here, add a new define for this mode and add it to the base .c files that needs it to enable the required functions